sdk/template-LLC/Azure.Template.LLC/README.md

```diff
@@ -72,32 +72,64 @@ If possible, use the same example snippets that your in-code documentation uses.
 
 Each example in the *Examples* section starts with an H3 that describes the example. At the top of this section, just under the *Examples* H2, add a bulleted list linking to each example H3. Each example should deep-link to the types and/or members used in the example.
 
-* [Create the data](#create-the-data)
-* [Get the data](#get-the-data)
-* [List the data](#list-the-data)
+* [Create resource](#create-resource)
+* [Get resource](#get-resource)
+* [List resources](#list-resources)
+* [Delete resource](#delete-resource)
+
+### Create resource
+
+Use the `Create` method to create a resource.
+
+```C# Snippet:CreateResource
+var client = GetClient();
+var resource = new
+{
+    name = "TeamplateResource",
+    id = "123",
+};
+Response response = await client.CreateAsync(RequestContent.Create(resource));
+using JsonDocument resourceJson = JsonDocument.Parse(response.Content.ToMemory());
+string resourceName = resourceJson.RootElement.GetProperty("name").ToString();
+string resourceId = resourceJson.RootElement.GetProperty("id").ToString();
+Console.WriteLine($"Name: {resourceName} \n Id: {resourceId}.");
+```
 
-### Create the data
+### Get resource
 
-Use the `CreateData` method to create a data reference.
+The `Get` method retrieves a data from the service. The `id` parameter is the unique ID of the data.
 
-```C#
-var data = client.CreateData(id, name);
+```C# Snippet:RetrieveResource
+var client = GetClient();
+var response = await client.GetAsync("123");
+using JsonDocument resourceJson = JsonDocument.Parse(response.Content.ToMemory());
+string resourceName = resourceJson.RootElement.GetProperty("name").ToString();
+string resourceId = resourceJson.RootElement.GetProperty("id").ToString();
+Console.WriteLine($"Name: {resourceName} \n Id: {resourceId}.");
 ```
 
-### Get the data
+### List resources
 
-The `GetData` method retrieves a data from the service. The `id` parameter is the unique ID of the data.
+The `GetResources` method retrieves the list of resources from the service.
 
-```C#
-var data = client.GetData("TestSecret");
+```C# Snippet:ListResources
+var client = GetClient();
+AsyncPageable<BinaryData> pageable = client.GetResourcesAsync();
+await foreach (var page in pageable.AsPages())
```

Comment:
This prints out the name and id of the first resource in each page, which is a relatively uncommon use case.  What if we did

```csharp
await foreach (var resource in resources)
```

instead?